.nvidia-ci.yml

```diff
@@ -244,3 +244,60 @@ release:ngc-packaging:
   extends:
     - .dist-packaging
     - .release:ngc
+
+# Define the external image signing steps for NGC
+# Download the ngc cli binary for use in the sign steps
+.ngccli-setup:
+  before_script:
+    - export NGCLI_VERSION=3.41.4
+    - apt-get update && apt-get install -y curl unzip
+    - curl -sSLo ngccli_linux.zip https://api.ngc.nvidia.com/v2/resources/nvidia/ngc-apps/ngc_cli/versions/${NGCLI_VERSION}/files/ngccli_linux.zip
+    - unzip ngccli_linux.zip
+    - chmod u+x ngc-cli/ngc
+    - export PATH=$(pwd)/ngc-cli:${PATH}
+    - NGC_CLI_API_KEY=${NGC_REGISTRY_TOKEN} ngc version upgrade --org nvidia -y
```

Comment:
Question: Should we rather set this as a variable to reduce the risk of leaking this on the command line?